tests/elements-regression/env-setup/actions/change-font-display.js

```diff
@@ -0,0 +1,26 @@
+const ElementorSettingsPage = require( '../../pages/elementor-settings-page' );
+
+let elementorFontDisplayOriginalValue;
+
+module.exports = {
+	beforeAll: async ( { page } ) => {
+		const settingsPage = new ElementorSettingsPage( page );
```

Comment:
Maybe just config it via `wp-cli` command?
https://github.com/elementor/elementor/blob/master/tests/wp-env/config/setup.sh#L10

Reply:
But then it will affect only the CI, what about local testing?

Reply:
It can be tested locally? (Linux screenshot)

Reply:
You can create your own photos + You can run on a container (e3 or `npm run test:elements-regression:update-snapshots`)